playground/engine.py

```diff
@@ -0,0 +1,818 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Now this is a library component. I would argue the module this installs with should have threedgrt in the name. This is asking for a name clash. Client code:

```
import playground.engine
```